README.md

```diff
@@ -1,11 +1,19 @@
 # Query Knowledge-base with Domain Specific Documents
 # work in progress
 
-There are many challenges involved when documents contain bunch of information stored in the form of tables and even images. One of the biggest challenge involved is to understand how the information is represented in the knowledge base(Graph). It's quite an ambiguous task to pick the entities from that domain, understand their type and determine the relationship between them. The Code pattern (`build knowledge-base with domain-specific documents`)[https://github.com/IBM/build-knowledge-base-with-domain-specific-documents] discusses the strategy to deal with this challenge.
+This code pattern is in continuation of the composite pattern - (`build knowledge-base with domain-specific documents`)[https://github.com/IBM/build-knowledge-base-with-domain-specific-documents]. We saw how we can extract entities and relations from a word document that contains information in tables and text to build a knowledge graph.
```

Comment:
the markdown syntax used here is reversed. you used `(x)[y]` it should be `[x](y)`, this won't render properly